reactExercise/solutions/zihe/ex6/src/hooks/useReFetch.js

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from "react"
+
+const useReFetch = (propsTimer) => {
+    const [loading, setLoading] = useState(true)
+    const [timer, setTimer] = useState(propsTimer * 1000)
+    const shouldReFetch = () => {
```

Comment:
shouldRefetch 这个名字一般会用return 一个 boolean,  因为 should => yes or no
```suggestion
    const Refetch= () => {
```